src/order/galois_connection.lean

```diff
@@ -6,27 +6,50 @@ Author: Johannes Hölzl
 import order.complete_lattice
 import order.order_iso
 /-!
-# Galois connections, Galois insertions and Galois coinsertions
+# Galois connections, insertions and coinsertions
+
```

Comment:
Maybe  explain somewhere what `u` and `l` stand for (I think it's upper and lower or something like this).